azurerm/resource_arm_key_vault.go

```diff
@@ -172,6 +188,13 @@ func resourceArmKeyVault() *schema.Resource {
 func resourceArmKeyVaultCreateUpdate(d *schema.ResourceData, meta interface{}) error {
 	client := meta.(*ArmClient).keyVaultClient
 	ctx := meta.(*ArmClient).StopContext
+
+	sku := expandKeyVaultSku(d)
+
+	if sku == nil {
```

Comment:
Same as mentioned above, lets pull the logic out here so its obvious what the end result is in 2.0

Reply:
K... moved logic...